graph/ui-icons/src/lib/technology-icon.tsx

```diff
@@ -0,0 +1,27 @@
+import { Framework, frameworkIcons } from './framework-icons';
+
+export interface TechnologyIconProps {
+  technology?: string;
+  showTooltip?: boolean;
+}
+
+export function TechnologyIcon({
+  technology,
+  showTooltip,
+}: TechnologyIconProps) {
+  if (!technology || !frameworkIcons[technology as Framework]) {
+    return null;
```

Comment:
We're not showing anything at the moment?

Can we please update this to show a generic element with the first character of the technology please?

Reply:
i updated the ui to be something like:
![Screenshot 2024-04-25 at 11 05 48 AM](https://github.com/nrwl/nx/assets/16211801/1c95ac1f-5a74-4dae-ad51-243539615252)
![Screenshot 2024-04-25 at 11 05 42 AM](https://github.com/nrwl/nx/assets/16211801/279d31d6-b949-4371-83b7-4e4fb42a67a4)